internal/service/wafv2/web_acl.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/aws/aws-sdk-go-v2/aws"
 	"github.com/aws/aws-sdk-go-v2/service/wafv2"
 	awstypes "github.com/aws/aws-sdk-go-v2/service/wafv2/types"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/structure"
```

Comment:
Should be `github.com/hashicorp/terraform-plugin-sdk/v2/helper/structure` or else we're pulling in the _very_ old Plugin SDK v1.